receiver/googlecloudspannerreceiver/factory_test.go

```diff
@@ -38,8 +38,8 @@ func TestType(t *testing.T) {
 
 func TestCreateMetricsReceiver(t *testing.T) {
 	factory := NewFactory()
-	cfg := factory.CreateDefaultConfig()
-	receiverConfig := cfg.(*Config)
+	config := factory.CreateDefaultConfig()
```

Comment:
Why has this been changed? This shadows the import `go.opentelemetry.io/collector/config` with no apparent reason.

Reply:
Made this by mistake. Corrected.